app/metrics.yaml

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+toolbar.events:
+  url_committed:
+    type: event
+    description: >
+      Recorded when a URL is committed from the toolbar.
+    bugs:
+      - 1539080
+    data_reviews:
+      - TBD
+    notification_emails:
+      - telemetry-client-dev@mozilla.com
```

Comment:
Can you add a different email here? This is meant to be for the Glean SDK team, so if this is for your team specifically, you should update this email to include a list that you will be on, and can be notified by.

In other bugs, this has been the Product manager (for example).

Reply:
@travis79 this can be a list with both your and my `@mozilla.com` addresses!

Reply:
Okay, I have updated this to my email and yours @Dexterp37.